### PR TITLE
Update ToStringValue.js

### PR DESCRIPTION
**Pull request by Ubachukwu-Ibeh**
Added 'bigint' to getToStringValue case

### CODE REVIEW

File: packages/react-dom/src/client/ToStringValue.js [hunk 1]
```diff
@@ -28,6 +29,7 @@ export function getToStringValue(value: mixed): ToStringValue {
     case 'number':
     case 'object':
     case 'string':
+    case 'bigint':
```
Comment (rheh) [inline packages/react-dom/src/client/ToStringValue.js:32]: nit-pick: Both the opaque type and case statement are ordered A-Z.  Thus, should be move bigint into its natural place? e.g.

```
    case 'bigint':
    case 'boolean':
    case 'number':
    case 'object':
    case 'string':
```

Comment (redbossrabbit) [reply packages/react-dom/src/client/ToStringValue.js:32]: Alright. Will do.

**Comment by rickhanlonii** [comment]
Could you add a test for this?

**Comment by redbossrabbit** [comment]
> Could you add a test for this?
Will do.

**Comment by redbossrabbit** [comment]
Will do.

**Review by rheh — APPROVED** [review]

**Pull request closed without merging**